### ISSUE TITLE
SCANsat parts do not work, no toolbar button

### ISSUE THREAD

**Issue by SquidDNA** [KSPModStewards/SCANsat#121]
Narrative: 
  After a decision to throw away some mods in favor of other ones, I removed all mods from my gamedata directory and did fresh downloads of everything I needed, then started a new career game. It's important to note that this resulted in me going from SCANsat 8.1 to 10. Partway into a mission to Mun I discovered that the SCANsat altimetry module wasn't doing anything when I right-clicked on it. I remembered that the usual part-interaction options were unavailable in VAB when I built the ship. I also noticed at this time that SCANsat was missing from the toolbar and was not available as something which could be loaded. After checking I had the latest version installed, I began removing other mods and reloading the game, and testing SCANsat by putting a command pod with an altimetry module on the launchpad in a new sandbox game. Even after all other mods had been removed, this consistently yielded a "dead module" which did not give a box when right-clicked on. (Right clicking on the pod yielded the usual results.) I also deleted the modulemanager cache file at intervals (on a hunch.) This did not help. 
  Finally deleted the Kerbal Space Program root directory, "deleted local content" via steam, and redownloaded the game. I then placed the top-level SCANsat directory into GameData along with ModuleManager.2.5.10 (had 2.5.9 previously), then tried testing SCANsat functionality with the lonely command pod as above. It still does not work. I did not test toolbar appearance here but I figured it was pointless.

tl;dr:
  SCANsat broke when I upgraded from 8.1 to 10.
  SCANsat parts are available for placement on ships and appear in the game correctly but otherwise do not do anything
  it's not because of any other mods
  I'm very sad

System information:
  Win7 SP1 64bit  (NOT 64bit KSP!)
  AMD Phenom II X4 965 2.4 GHz
  8GB RAM
  GeForce GTX 460 v2 (driver 347.52)
  2-monitor setup, stretched desktop, problem occurs whether KSP is fullscreen or windowed.

Thanks in advance for your help.

**Comment by DMagic1** [comment]
I need your output_log file from the KSP_Data folder to be able to help you out.

**Comment by SquidDNA** [comment]
Too long to paste, can't attach the file, so here: http://pastebin.com/pREEwaJB 

Thanks for having a look.

**Comment by DMagic1** [comment]
You downloaded the source file package, not the release package.

Get the release package in the [releases](https://github.com/S-C-A-N/SCANsat/releases/tag/v10.0) section here, at the very bottom select the SCANsat_v10.zip, not the source code zip.

Or you can get it at [KerbalStuff](https://kerbalstuff.com/mod/249).

**Comment by SquidDNA** [comment]
How embarassing; thank you. As I recall I downloaded this by clicking the "Download ZIP" link on the right hand side of the page, which just seemed like the thing to do. This probably also explains why Targetron was not working.
